development/kops/run_all.sh

```diff
@@ -13,26 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -exo pipefail
+BASEDIR=$(dirname "$0")
+
 function cleanup()
 {
   echo 'Deleting...'
-  ./delete_cluster.sh || true
-  ./delete_store.sh
+  ${BASEDIR}/delete_cluster.sh || true
+  ${BASEDIR}/delete_store.sh
   exit 255;
 }
 
-set -exo pipefail
-BASEDIR=$(dirname "$0")
 echo "This script will create a cluster, run tests and tear it down"
 cd "$BASEDIR"
```

Comment:
Not sure why all these changes are needed when we cd to the BASEDIR here

Reply:
Removed the cd call.

Reply:
Terry makes a good point here, adding the BASEDIR everyone does seem to clutter things up, if we were cd-ing why was this necessary?  Is there another way we could accomplish this?

Reply:
I changed it back to just ./<script>. Usually I prefer to avoid cd-ing into directories in shell script.s It becomes problematic while adding/modifying the down level scripts. But in this case we are cd-ing only once. So, I guess it is okay.

Reply:
I wish we'd written this in go. It became a monster and design wise it would be way more manageable in something other than bash